src/context.ts

```diff
@@ -115,7 +115,7 @@ export class Context<E extends WebhookPayloadWithRepository = any> implements We
   public issue<T> (object?: T) {
     const payload = this.payload
     return Object.assign({
-      number: (payload.issue || payload.pull_request || payload).number
+      issue_number: (payload.issue || payload.pull_request || payload).number
```

Comment:
question: Do want want a separate `context.pull` to complement `context.issue` now that the API formats differ?

Reply:
huh, interesting point. I guess so? /cc @probot/maintainers

Reply:
> question: Do want want a separate `context.pull` to complement `context.issue` now that the API formats differ?

Okay, let’s do it 👍

Reply:
> Okay, let’s do it 👍

Yes! I’m gonna be unavailable for the rest of the week (celebrating my wife’s birthday :tada:) but should be able to look at this next week. Hopefully it can wait till then :bow:

Reply:
It sure can 👍 happy celebrating!